migrations/sqls/20190512164549-services-list-up.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE services MODIFY COLUMN serviceDesc VARCHAR(1028);
+
+INSERT INTO services (
```

Comment:
Vicky is working on the descriptions for the services -- I'll create another db migration to update the service descriptions once she gives me them.